scripts/pi-hole/js/index.js

```diff
@@ -760,17 +766,17 @@ $(document).ready(function() {
                         var m = parseInt(time[2], 10) || 0;
                         var from = padNumber(h)+":"+padNumber(m-5)+":00";
                         var to = padNumber(h)+":"+padNumber(m+4)+":59";
-                        return "Upstreams from "+from+" to "+to;
+                        return "Queries from "+from+" to "+to;
                     },
                     label: function(tooltipItems, data) {
-                        if(tooltipItems.datasetIndex === 1)
+                        if(tooltipItems.datasetIndex === 0)
```

Comment:
Percentage should be appended to the blocked queries (not dataset 0, was 1 before)